frontend/src/services/api.js

```diff
@@ -0,0 +1,224 @@
+const API_WEBSITE = 'https://skincare-api.herokuapp.com'
+
+const BACKUP_PRODUCTS = [
```

Comment:
Can you fetch these products from an API or create a database storage, and store the product data in a database, such as MySQL or MongoDB. This allows you to easily manage and update the product information using CRUD (Create, Read, Update, Delete) operations